app/mailers/sac_cas/application_mailer.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module SacCas::ApplicationMailer
+  private
+
+  def compose(recipients, content_key, locales = [])
+    return if recipients.blank?
+
+    values = values_for_placeholders(content_key)
+    custom_content_mail(recipients, content_key, values, {}, locales)
+  end
+
+  def custom_content_mail(recipients, content_key, values, headers = {}, locales = [])
+    content = CustomContent.get(content_key)
+    locales = [I18n.locale] if locales.empty?
+    headers[:to] = use_mailing_emails(recipients)
+
+    contents = locales.map do |locale|
+      I18n.with_locale(locale) do
+        body, subject = content_subject_and_body(content, values, locale, locales)
+
+        if body.body.nil?
```

Comment:
This looks strange. Why is `body.body` needed?

Reply:
I ran the specs and it isn't covered, so either not needed or not tested.

Reply:
> This looks strange. Why is `body.body` needed?

`body.blank?` does the same 👍